src/PoolManager.sol

```diff
@@ -377,15 +377,15 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, Claims {
     }
 
     /// @inheritdoc IPoolManager
-    function mint(Currency currency, address to, uint256 amount) external noDelegateCall onlyByLocker {
+    function mint(address to, Currency currency, uint256 amount) external override noDelegateCall onlyByLocker {
```

Comment:
why switch the ordering here?

Reply:
this matches the underlying 6909 mint/burn functions